### ISSUE TITLE
Delete Work Item ID column in Service and Product Profile tables once the new Work Item design gets implementated

### ISSUE THREAD

**Issue by maririos** [Azure/azure-sdk-tools#6551]
Once we finish issue https://github.com/Azure/azure-sdk-tools/issues/6453 we should remove the `Work Item ID` column from Service and Product profiles tables, since it's no longer going to be used in the future design.

**Comment by ladonnaq** [comment]
@maririos Is this still valid?

**Comment by maririos** [comment]
Yes. It doesn't affect our users. It is a technical detail on how the relationships across our tables in data verse work.
